react/features/base/participants/reducer.js

```diff
@@ -356,12 +358,15 @@ ReducerRegistry.register('features/base/participants', (state = DEFAULT_STATE, a
 /**
  * Returns the participant's display name, default string if display name is not set on the participant.
  *
+ * @param {Object} state - The local participant redux state.
  * @param {string} name - The display name of the participant.
  * @returns {string}
  */
-function _getDisplayName(name) {
-    return name
-        ?? (typeof interfaceConfig === 'object' ? interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME : 'Fellow Jitser');
+function _getDisplayName(state, name) {
+    // $FlowExpectedError
```

Comment:
What does flow complain about here?

Reply:
Cannot get state['features/b...'] because property features/base/config is missing in object literal [1].

Reply:
Doesn't adding `:Object` to the function signature help?

Reply:
Let me try